opossum-tool/OpossumUI
# Add bash script to check out a PR and directly open a file in OpossumUI

### Summary of changes

This PR adds a script to make development easier. 
```
Usage: ./tools/checkout_PR_and_start_opossum_ui.sh [OPTIONS]
Options:
 -p, Set the number of the PR to use
 -c, Set the commit hash to use
 -n, Set the name of the remote, default: origin
 -f, Set absolute path to file which should be opened
 -o, Checkout only, don't open the app
Note: either use -c or -p
Note: The script assumes to be run in the root directory of a local clone of the original repo

```

### Context and reason for change

This script should help developers to test changes on large files,

### How can the changes be tested
Play around with the script.
Note that you need to store a duplicate of the script (for example change the name and don't add it to Git afterwards) as the script will check out a different PR where this script no  longer exists. Run e.g.
`./tools/checkout_PR_and_start_opossum_ui.sh -p 2359 -f ./example-files/opossum_input.json` . See that the respective PR is checked out and the OpossumUI is opened with the provided file.
